src/main/java/seedu/address/logic/Logic.java

```diff
@@ -42,6 +43,9 @@
     /** Returns an unmodifiable view of the filtered list of quests */
     ObservableList<Quest> getFilteredQuestList();
 
+    /** Returns an unmodifiable view of the filtered list of consultations */
+    ObservableList<Consultation> getFilteredConsultationsList();
```

Comment:
was this the problem? I'm still confused as to why I had the error in the first place XD

Reply:
Yeah this was the problem, i wrote consultations and you wrote consultation haha